pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -104,8 +104,10 @@ type ReconcileKnativeServing struct {
 
 // Create manifestival resources and KnativeServing, if necessary
 func (r *ReconcileKnativeServing) InjectClient(c client.Client) error {
-	m, err := mf.NewManifest(*filename, *recursive, c)
+	koDataDir := os.Getenv("KO_DATA_PATH")
+	m, err := mf.NewManifest(filepath.Join(koDataDir, "knative-serving-0.6.0.yaml"), *recursive, c)
 	if err != nil {
+		log.Error(err, "Failed to load manifest")
 		return err
```

Comment:
Plz move deploy/crds/serving_v1alpha1_knativeserving_cr.yaml to  cmd/manager/kodata/ as well.

And change func (r *ReconcileKnativeServing) ensureKnativeServing() (err error) in this file.